src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_DashboardStack/Dashboard_TripSearchScreen/use-flexible-transport-direct-mode-enabled.ts

```diff
@@ -0,0 +1,19 @@
+import {useRemoteConfig} from '@atb/RemoteConfigContext';
+import {StorageModelKeysEnum} from '@atb/storage';
+import {useDebugOverride} from '@atb/debug';
+
+export const useFlexibleTransportDirectModeEnabled = () => {
+  const [debugOverride] = useFlexibleTransportDirectModeDebugOverride();
+  const {use_flexible_on_directMode: enabledInRemoteConfig} = useRemoteConfig();
+
+  if (debugOverride !== undefined) {
```

Comment:
I see the following code part is duplicated in 3 or more files. Could it go in its own hook?
`if (debugOverride !== undefined) {
    return debugOverride;
  }
  return enabledInRemoteConfig;`

Or alternatively combine the ones you have and return multiple things.

Reply:
@hanne-at-atb I made a code refactor please take a look

Reply:
Nice!